### ISSUE TITLE
Connections to Frontend for NFT Information

### ISSUE THREAD

**Issue by Noah-Kurz** [amosproj/amos2022ss07-nft-playbook#40]
## User story
1. As a poweruser
2. I want / need easy connections
3. So that I can easily extend the tool

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
